test/functorch/test_aotdispatch.py

```diff
@@ -35,7 +35,7 @@
 from torch.testing._internal.common_methods_invocations import op_db
 from torch.testing._internal.common_modules import module_db, modules
 from torch.testing._internal.common_utils import parametrize, instantiate_parametrized_tests
-from torch.testing._internal.control_flow_opinfo_db import control_flow_opinfo_db
+from torch.testing._internal.hop_opinfo_db import control_flow_opinfo_db
```

Comment:
why not just use the hop_db instead of `control_flow_opinfo_db`? All HOPs should pass these tests too.